src/utils/sendUserDiscordData.ts

```diff
@@ -0,0 +1,37 @@
+import { env } from "../typeDefinitions/default.types";
+import jwt from "@tsndr/cloudflare-worker-jwt";
+import { RDS_STAGING_API_URL } from "../constants/urls";
+
+export const sendUserDiscordData = async (
+  token: string,
+  discordId: number,
+  env: env
+) => {
+  const authToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 15 },
+    env.DISCORD_PRIVATE_KEY,
```

Comment:
Can we please create tickets for adding this environment variable to the staging and production environment

Reply:
I already have created ticket for it 
https://github.com/Real-Dev-Squad/discord-slash-commands/issues/36 
Please have a look and let me know if it requires any changes